src/actions.js

```diff
@@ -41,6 +42,12 @@ exports.drawCard = playerId => (dispatch, getState) => {
   }
 };
 
+exports.freeze = (frozenFor, ...ids) => ({
+  type: FREEZE,
+  frozenFor,
+  ids,
```

Comment:
use `characterIds` instead of `ids`. In hearthstone lingo a character is a minion or a hero.